_posts/2016-10-11-your-community-manager.md

```diff
@@ -0,0 +1,53 @@
+---
+name: your-community-manager
+layout: post
+title: "Greetings from Your Community Manager!"
+date: 2016-10-11
+authors:
+  - name: Stefanie Butland
+    url: https://ropensci.org/about/
+tags:
+- ropensci
+- community
+- jobs
+
+---
+
+I feel both proud and privileged to join rOpenSci as your Community Manager. I’ve been a compulsive community builder since the early 2000’s, but it has rarely been part of my job description. Now it seems like all roads have led to this. After a couple of fine days of indoctrination at the [UC Berkeley](https://bids.berkeley.edu/) home of rOpenSci, I’m settled into work in beautiful Kamloops, British Columbia, Canada.
+
+So much of my perspective of rOpenSci comes from being a newcomer. I was impressed with the funding from three major grants (you must be doing a few things right!), [@ropensci](https://twitter.com/rOpenSci) having nearly 11,000 followers on Twitter, the awesome [staff, leadership team and advisory board](https://ropensci.org/about/#staff), growing from one full time person to four (with two more positions to open up), and having an enthusiastic [community](https://ropensci.org/community/) that is known for helping each other out and getting things done.
+
+My Open Science Epiphany
+-----------------------------------------------------------------
+
+I completed my MSc in Biology at York University in Toronto, doing fruit fly behaviour genetics in a lab with a strong foundation in ecology and evolution and I’ve done molecular biology-based research in bacteria, plants, insects and mammals. My open science epiphany came in the late 90’s when I found so many different sequences of my favourite enzyme in public databases that I could learn more and faster by comparing these than I could by slowly cloning and sequencing one more example!
+
+You can check out my science [here](https://scholar.google.ca/citations?user=9HnWx2kAAAAJ).
```

Comment:
this sentence feels like it's hanging a bit awkwardly. what about attaching to previous paragraph?  

also, instead of `[here]` showing up for readers, maybe say `[at Google Scholar]` or `[check out my science]`